unitylibs/core/workflow/workflow-acrobat/service-handler.js

```diff
@@ -15,10 +15,16 @@ export default class ServiceHandler {
   }
 
   getHeaders() {
+    const gat = getGuestAccessToken();
+    if (!gat) {
+      const error = new Error();
+      error.message = 'imsError';
+      throw error;
+    }
```

Comment:
Nit: You can do this in 1 line
```suggestion
    if (!gat) throw Object.assign(new Error(), { message: 'imsError' });
```